### ISSUE TITLE
[CLI] Cannot disable boolean flags

### ISSUE THREAD

**Issue by Derppening** [GZoltar/gzoltar#12]
### Context
When using the Command-Line Tool to create a fault localization report, the user cannot specify to disable the boolean values of the `--incl*` family.

### Steps to Reproduce
1. Construct a fault localization report from any instrumentation file, using any of the `--incl` flags and a boolean value.
2. Notice that the CLI returns `No argument is allowed: false`

### GZoltar arguments
I used Gradle's `JavaExec` task to invoke GZoltar, which is equivalent to the Command-Line invocation of:
```
java -cp $mainClasses:$gzoltarJars com.gzoltar.cli.Main faultLocalizationReport --buildLocation $buildLocation --granularity line --inclStaticConstructors false --dataFile $dataFile --outputDirectory $output --family sfl --formula ochiai --metric entropy --formatter txt
```
Where the `$`-variables correspond to my project-specific variables and paths.

### Expected behaviour
I should be able to disable the `--incl*` flags using, for example `--inclDeprecatedMethods false`.

### Environment (please complete the following information, if relevant):
 - OS: Linux
 - Version 1.7.2

### Additional comments
According to [Option4J's documentation](https://args4j.kohsuke.org/args4j/apidocs/org/kohsuke/args4j/Option.html), it states that

> When T is boolean , it represents a boolean option that takes the form of -OPT. When this option is set, the property will be set to true.

In other words, I believe that the fix is either to set the `@Option` booleans to false, or replace the `--incl*` include flags with `--excl*` exclude flags.

**Comment by jose** [comment]
Hi @Derppening,

Thank you for reporting this issue.

> I believe that the fix is either to set the @Option booleans to false

Yes, agree.


--
Best,
Jose
